recipes/oifits/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "oifits" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 611d8ef9040aabf9db87e9419a25be18350f59341771587dde3d6d3cafc46f48
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
this isn't noarch if it needs a compiler

Reply:
Thanks @scopatz ! I fixed it!